deploy/20_cloud-ingress-operator_trusted-ca-bundle.ConfigMap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Does this configmap get deployed via the OLM bundle?

Reply:
That's correct, it does. I've done some tests to verify that OLM is successfully:
- installing it (via a freshly-built and deployed catalog image)
- not clobbering it after cluster-network-operator injects the CAs into it
- cleaning it up if the CSV is removed